Analysis/EventFiltering/centralEventFilterProcessor.cxx

```diff
@@ -0,0 +1,185 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// @file   centralEventFilterProcessor.cxx
+
+#include "centralEventFilterProcessor.h"
+
+#include "Framework/ControlService.h"
+#include "Framework/ConfigParamRegistry.h"
+#include "Framework/TableConsumer.h"
+
+#include <TFile.h>
+#include <TH1D.h>
+
+#include <iostream>
+#include <cstdio>
+#include <fmt/format.h>
+#include <rapidjson/document.h>
+#include <rapidjson/filereadstream.h>
+
+using namespace o2::framework;
+using namespace rapidjson;
+
+namespace
+{
+Document readJsonFile(std::string& config)
+{
+  FILE* fp = fopen(config.data(), "rb");
+
+  char readBuffer[65536];
+  FileReadStream is(fp, readBuffer, sizeof(readBuffer));
+
+  Document d;
+  d.ParseStream(is);
+  fclose(fp);
+  return d;
+}
+} // namespace
+
+namespace o2::aod::filtering
+{
+
+void CentralEventFilterProcessor::init(framework::InitContext& ic)
+{
+  // JSON example
+  // {
+  //   "subwagon_name" : "CentralEventFilterProcessor",
+  //   "configuration" : {
+  //     "NucleiFilters" : {
+  //       "H2" : 0.1,
+  //       "H3" : 0.3,
+  //       "HE3" : 1.,
+  //       "HE4" : 1.
+  //     }
+  //   }
+  // }
+  std::cout << "Start init" << std::endl;
+  Document d = readJsonFile(mConfigFile);
```

Comment:
Why do you need a separate mechanism than the `Configurables`?

Reply:
If `Configurables` support nested maps we can use them